### PR TITLE
Fixing openshift-jenkins-master.yaml to use dockerfile from ocp-build-data

### PR DESCRIPTION
**Pull request by alvaroaleman**

### CODE REVIEW

**Comment by jupierce** [comment]
Since at least one of these images is based on the cli, I don't believe we can get this green until the cli is migrated to RHEL8.

File: ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ images:
     src:
       paths:
       - destination_dir: .
-        source_path: /go/src/github.com/openshift/jenkins/agent-nodejs-10/.
+        source_path: /go/src/github.com/openshift/jenkins/agent-nodejs-8/.
```
Comment (yselkowitz) [inline ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml:42]: This should be agent-nodejs-12 instead, with a dockerfile_path of Dockerfile.rhel8.

**Pull request merged**